src/search/verification/formal_verifier_equiv/src/lib.rs

```diff
@@ -461,6 +464,14 @@ pub fn rules(mut nums: Vec<u32>) -> Vec<Rewrite> {
 
     ].concat();
 
+    let mut rules9 = vec![
+
+        rw!("bc-div-commute-partition";
+            "(bc_div (partition ?t0 ?d0 ?d1 ?i0) ?t1)"
```

Comment:
This commutativity only applies to the broadcast dimension (the inner most dimension)